shared/actions/tracker.js

```diff
@@ -171,9 +171,20 @@ export function registerIdentifyUi (): TrackerActionCreator {
     }
 
     engine().setIncomingHandler('keybase.1.identifyUi.delegateIdentifyUI', (param: any, response: ?Object) => {
+      // If we don't finish the session by our timeout, we'll display an error
+      const trackerTimeout = 1e3 * 60 * 5
+      let trackerTimeoutError = 0
+
+      const onStart = (username) => {
+        trackerTimeoutError = setTimeout(() => {
+          dispatch({type: Constants.identifyFinished, error: true, payload: {username, error: 'Identify timed out'}})
+        }, trackerTimeout)
+      }
+
       const session: Session = engine().createSession(
-        serverCallMap(dispatch, getState, false, () => {
+        serverCallMap(dispatch, getState, false, onStart, () => {
```

Comment:
:sheep: Maybe move this arrow function out into an `onFinish` const to match `onStart`